### PR TITLE
[mds-policy] fix heisenbug by moving tests to end

### PR DESCRIPTION
**Pull request by karcass**
## Impacts
- [x] Policy

### CODE REVIEW

File: packages/mds-policy/api.ts [hunk 1]
```diff
@@ -94,9 +94,11 @@ function api(app: express.Express): express.Express {
   })
 
   app.get(pathsFor('/policies/:policy_id'), async (req, res) => {
+    log.info('read policy', JSON.stringify(req.params))
     const { policy_id } = req.params
     if (!isUUID(policy_id)) {
       res.status(400).send({ error: 'bad_param' })
```
Comment (drtyh2o) [inline packages/mds-policy/api.ts]: Most other places we just `return res.status(...)` in a single statement.

**Comment by avatarneil** [comment]
FWIW the issue also appears in the `policy-author` tests

**Comment by karcass** [comment]
`policy-author` passes 100% of time for me after like 50 attempts.  Can you PM me the output of a failure?

**Comment by karcass** [comment]
I'm kinda curious too but digging deeper at this time didn't seem like the best use of my attention.

**Comment by avatarneil** [comment]
I don't have one handy -- they're probably lost in codeship because we've run re-builds. But I'm a good 90% sure we've seen them there, too, when trying to read a policy back.

**Pull request merged**